research/src/pages/breadcrumb.proposal.mdx

```diff
@@ -0,0 +1,72 @@
+---
+menu: Components
+name: Breadcrumb (Editor's Draft)
+path: /components/breadcrumb
+pathToResearch: /components/breadcrumb.research
+---
+
+import BreadcrumbAnatomy from '../components/breadcrumb-anatomy'
+
+## Overview
+
+Breadcrumbs are navigation land marks used to show a user's location within the site or app.
+
+## Use Cases
+
+A `<breadcrumb />` trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place within a website or web application. Breadcrumbs are often placed horizontally before a page's main content.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/breadcrumb/)
+- [Auth0](https://styleguide.auth0.com/components/breadcrumb)
+- [Atlas Kit](https://atlassian.design/components/breadcrumbs/examples)
+- [Carbon](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+- [FAST](https://explore.fast.design/components/fast-breadcrumb)
+- [Lightning](https://www.lightningdesignsystem.com/components/breadcrumbs/)
+- [Semantic UI](https://semantic-ui.com/collections/breadcrumb.html)
+
+### Anatomy <a class="link" href="#anatomy" id="anatomy"></a>
+
+#### Diagram <a class="link" href="#diagram" id="diagram"></a>
+
+<BreadcrumbAnatomy />
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<breadcrumb>` - The root element that that has the semantic value of navigation **[required]**
+- `<ordered-list>` - The list containing items and dividers **[required]**
+- `<list-item>` - The wrapper to items or divider content **[required]**
+
+Proposed HTML:
+
+```html
+<nav>
+  <ol>
+    <li><a>Item 1</a></li>
+    <li aria-hidden="true">/</li>
+    <li><a>Item 2</a></li>
+  </ol>
+</nav>
+```
+
+#### Accessibility <a class="link" href="#accessibility" id="accessibility"></a>
+
+The Breadcrumb is composed by:
+
+- A `nav` equivalent wrapper that should always have `aria-label` or `aria-labelledby`
```

Comment:
this would only be necessary if, when implemented, a `nav` was not given more context to its purpose.

I'd submit that there's no reason to add this, and as a web component an accessible name of `breadcrumb` or an `aria-roledescription` of `breadcrumb` could instead be set by default to not require an author need to specify these